float32 finger_vel
float32 wrist_vel
float32 shoulder_vel